CppCoreGuidelines.md

```diff
@@ -7940,7 +7940,20 @@ Casting to a reference expresses that you intend to end up with a valid object,
 
 ##### Example
 
-    ???
+        class A { ... };
+
+        class B { ... };
+        
+        class C : public A, public B { ... };
+        
+        void some_function(B a) {
+          A base = dynamic_cast<A&>(a); // throws std::bad_cast
+        }
+        
+        void f() {
+          C some_object = C();
+          some_function(some_object);
```

Comment:
slicing intentional? Should be noted (and also, goes against ES.63 "Don't slice")